package/index.html

```diff
@@ -0,0 +1,2 @@
+test file
```

Comment:
I'm not sure what this file is for. Can you drop it from your PR?

Reply:
Yeah no problem.

Reply:
If you can delete this file and add/commit/push your changes to your branch then I'll approve the PR and merge it.

Reply:
Alright.